docs/en_US/Compression/v2_pruning_config_list.rst

```diff
@@ -0,0 +1,66 @@
+Pruning Config List Interpretation
+==================================
+
+The Keys in Config List
+-----------------------
+
+Each sub-config in the config list is a dict, and the scope of each setting (key) is only internal to each sub-config.
+If multiple sub-configs are configured with the same module, the later ones will overwrite the previous ones.
+
+op_types
+^^^^^^^^
+
+The type of the module specified in this sub-config.
+If ``op_names`` is not set in this sub-config, all modules in the model that satisfy the type will be selected.
+If ``op_names`` is set in this sub-config, the selected module should satisfy both type and name.
+
+op_names
+^^^^^^^^
+
+The name of the module specified in this sub-config.
+If ``op_types`` is set in this sub-config, the selected module should satisfy both type and name.
+
+op_partial_names
+^^^^^^^^^^^^^^^^
+
+This key is for the modules to be pruned with similar names. NNI will find all names in the model,
```

Comment:
here "similar" means matching prefix, or matching sub-string, or others?

Reply:
means `matching same sub-string`, will fix it

Reply:
fixed